$OpenBSD: patch-configure,v 1.2 2011/01/17 00:16:52 jasper Exp $

checking whether build environment is sane... ./configure[1808]: syntax error: `&' unexpected

and set correct Python version

--- configure.orig	Sun Jan 16 23:27:14 2011
+++ configure	Sun Jan 16 23:27:58 2011
@@ -1825,14 +1825,6 @@ echo timestamp > conftest.file
 # name.  Accept space and tab only in the latter.
 am_lf='
 '
-case `pwd` in
-  *[\\\"\#\$\&\'\`$am_lf]*)
-    as_fn_error $? "unsafe absolute working directory name" "$LINENO" 5 ;;
-esac
-case $srcdir in
-  *[\\\"\#\$\&\'\`$am_lf\ \	]*)
-    as_fn_error $? "unsafe srcdir value: \`$srcdir'" "$LINENO" 5 ;;
-esac
 
 # Do `set' in a subshell so we don't clobber the current shell's
 # arguments.  Must try -L first in case configure is actually a
@@ -2271,7 +2263,7 @@ if test "${am_cv_pathless_PYTHON+set}" = set; then :
   $as_echo_n "(cached) " >&6
 else
 
-	for am_cv_pathless_PYTHON in python python2 python3 python3.0 python2.5 python2.4 python2.3 python2.2 python2.1 python2.0 none; do
+	for am_cv_pathless_PYTHON in python${MODPY_VERSION} none; do
 	  test "$am_cv_pathless_PYTHON" = none && break
 	  prog="import sys
 # split strings by '.' and convert to numeric.  Append some zeros
